todo_app/app.py

```diff
@@ -1,20 +1,63 @@
 from flask import Flask, render_template, session, request, redirect
 from todo_app.flask_config import Config
 from todo_app.data.session_items import _DEFAULT_ITEMS, get_items, add_item
+import requests
+import os
 
 app = Flask(__name__)
 app.config.from_object(Config)
-
+# TODO a test that the environment has these values could be handy
+TRELLO_KEY = os.environ.get('TRELLO_KEY')
+TRELLO_TOKEN = os.environ.get('TRELLO_TOKEN')
+TRELLO_LIST_ID_DOING = os.environ.get('TRELLO_LIST_ID_DOING')
+TRELLO_LIST_ID_TODO = os.environ.get('TRELLO_LIST_ID_TODO')
+TRELLO_LIST_ID_DONE = os.environ.get('TRELLO_LIST_ID_DONE')
 
 @app.route('/')
 def index():
-    return render_template('index.html', items=get_items())
+    todos = requests.get(f"https://trello.com/1/lists/{TRELLO_LIST_ID_TODO}/cards?key={TRELLO_KEY}&token={TRELLO_TOKEN}")
+    dones = requests.get(f"https://trello.com/1/lists/{TRELLO_LIST_ID_DONE}/cards?key={TRELLO_KEY}&token={TRELLO_TOKEN}")
+    # loop through these two lists and create an instance of Item for each element
+    todos = itemizeList(todos.json())
+    dones = itemizeList(dones.json())
```

Comment:
Note that the Module 3 exercise assumes you get a single a list of all of the cards and then asks you to sort them into separate lists inside a "ViewModel".

You're already getting your items in separate lists so the exercise's suggestion doesn't make a lot of sense.

You probably want to just ignore that part but the stretch goal after it would still give you some practice writing unit tests. E.g. adding a "recent_done_items" property to give you all of the cards that were moved to "done" today.